### ISSUE TITLE
ts file containing global in its filename is renamed globalThis while SSR

### ISSUE THREAD

**Issue by BenLune** [analogjs/analog#660]
### Please provide the environment you discovered this bug in.

"nx": "^16.7.0",
"@analogjs/content": "^0.2.0",
"@analogjs/router": "^0.2.0",

### Which area/package is the issue in?

create-analog

### Description

I have this error :  Error: Failed to load url ./globalThis-base-entity.store (resolved id: ./globalThis-base-entity.store)
My file is named :  global-base-entity.store
If I rename it, no problem.

### Please provide the exception or error you saw

```true
Error: Failed to load url ./globalThis-base-entity.store (resolved id: ./globalThis-base-entity.store)
```

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [ ] No

**Comment by rlmestre** [comment]
Can you try the latest version (0.2.5)? Several related fixes were merged recently

**Comment by brandonroberts** [comment]
This was resolved in the `0.2.6` release
